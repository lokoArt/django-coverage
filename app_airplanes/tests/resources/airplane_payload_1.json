{
  "airplanes": [
    {
      "id": 100,
      "passengers_number": 100
    },
    {
      "id": 50,
      "passengers_number": 75
    },
    {
      "id": 70,
      "passengers_number": 87
    },
    {
      "id": 98,
      "passengers_number": 123
    },
    {
      "id": 1000,
      "passengers_number": 215
    },
    {
      "id": 1111,
      "passengers_number": 15
    },
    {
      "id": 155,
      "passengers_number": 31
    },
    {
      "id": 654,
      "passengers_number": 99
    },
    {
      "id": 12315,
      "passengers_number": 64
    },
    {
      "id": 8888,
      "passengers_number": 400
    }
  ]
}