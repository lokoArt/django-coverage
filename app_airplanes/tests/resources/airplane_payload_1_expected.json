[
  {
    "id": 100,
    "fuel_consumption": "1.800",
    "maximum_duration": 11111
  },
  {
    "id": 50,
    "fuel_consumption": "1.509",
    "maximum_duration": 6626
  },
  {
    "id": 70,
    "fuel_consumption": "1.650",
    "maximum_duration": 8484
  },
  {
    "id": 98,
    "fuel_consumption": "1.839",
    "maximum_duration": 10658
  },
  {
    "id": 1000,
    "fuel_consumption": "2.830",
    "maximum_duration": 70671
  },
  {
    "id": 1111,
    "fuel_consumption": "2.467",
    "maximum_duration": 90084
  },
  {
    "id": 155,
    "fuel_consumption": "1.814",
    "maximum_duration": 17086
  },
  {
    "id": 654,
    "fuel_consumption": "2.450",
    "maximum_duration": 53377
  },
  {
    "id": 12315,
    "fuel_consumption": "3.400",
    "maximum_duration": 724337
  },
  {
    "id": 8888,
    "fuel_consumption": "3.959",
    "maximum_duration": 448997
  }
]